ui/raidboss/data/03-hw/dungeon/the_lost_city_of_amdapor_hard.ts

```diff
@@ -0,0 +1,212 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export interface Data extends RaidbossData {
+  earthResistDown: { [name: string]: boolean };
+  windResistDown: { [name: string]: boolean };
+}
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.TheLostCityOfAmdaporHard,
+  timelineFile: 'the_lost_city_of_amdapor_hard.txt',
+  // Temporarily out of combat during Kuribu phases.  @_@;;
+  resetWhenOutOfCombat: false,
+  initData: () => {
+    return {
+      earthResistDown: {},
+      windResistDown: {},
+    };
+  },
+  triggers: [
+    {
+      id: 'LostCityHard Gremlin Bad-Mouth',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '775', source: 'Ranting Ranks Gremlin' }),
+      condition: Conditions.targetIsNotYou(),
+      infoText: (data, matches, output) => output.comfort!({ name: data.ShortName(matches.target) }),
+      outputStrings: {
+        comfort: {
+          en: '/comfort ${name}',
+        },
+      },
+    },
+    {
+      id: 'LostCityHard Achamoth Neuro Squama',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '15C5', source: 'Achamoth', capture: false }),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Go to edge; look outside',
+        },
+      },
+    },
+    {
+      id: 'LostCityHard Achamoth Toxic Squama',
+      type: 'HeadMarker',
+      netRegex: NetRegexes.headMarker({ id: '0001' }),
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Away from orb',
```

Comment:
Would it be possible to reword this as "Avoid chasing orb" or something similar, and call it for everyone? It shouldn't be too noisy in this encounter.